src/main/java/com/thealgorithms/datastructures/queues/PriorityQueues.java

```diff
@@ -30,37 +32,94 @@ class PriorityQueue {
      * @param size Size of the queue
      */
     public PriorityQueue(int size) {
-        maxSize = size;
-        queueArray = new int[size];
+        /* If capacity is not defined, default size of 11 would be used
```

Comment:
Where is this 11?

Reply:
I was going to add a default constructor but I guess I forgot at that time, Now I've added it